db/src/schema.rs

```diff
@@ -11,7 +11,7 @@
 #![allow(dead_code)]
 
 use entids;
-use errors::*;
+use errors::{ErrorKind, Result};
```

Comment:
Worth noting that `error-chain` explicitly suggests `use *` here; but I'd rather we're happy, so keep it.